src/translations/screens/FareContract.ts

```diff
@@ -218,4 +218,43 @@ export default orgSpecificTranslations(FareContractTexts, {
       ),
     },
   },
+  fram: {
+    details: {
+      barcodeErrors: {
+        notInspectableDevice: {
+          wrongDevice: (deviceName: string) =>
+            _(
+              `Du bruker billetter på din mobil, "${deviceName}". Husk å ta den med deg når du reiser.\nDu kan alltid bytte til reisekort eller en annen mobil ved å gå til **Min profil**.`,
```

Comment:
Should this be "Min bruker" and not "Min profil"? 🤔 Since `fram.navigation.profile` is changed in `src/translations/dictionary.ts`

Reply:
Good catch! Must've missed this one. 

I went ahead and used "user" based on FRAM's requests for the webshop, using user instead of profile there.

Reply:
I think that is good! Nice 👍